pnpm-lock.yaml

```diff
@@ -11,6 +11,7 @@ importers:
       axios: ^0.21.1
       cz-conventional-changelog: ^3.0.2
       eslint: 7.5.0
+      git-cz: ^4.7.6
```

Comment:
it is this dependency that allows emojis on your commit?

Reply:
[yep lol](https://www.npmjs.com/package/git-cz)

Reply:
Apparently yea! i absolutely dont know why tho haha but i had to download it in order to commit sooo...